aws_lambda_powertools/utilities/data_classes/common.py

```diff
@@ -65,7 +65,7 @@ def body(self) -> Optional[str]:
     @property
     def json_body(self) -> Any:
```

Comment:
Should we fix the older type annotation you Dict[str, Any] here too?

Reply:
It could be a list too :)